ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -252,6 +254,18 @@ Item {
             }
         }
 
+
+        Connections {
```

Comment:
WDYT about adding a nim property `sendingInProgress`and proxy it through store? Or even a store variable connected to nim signals. 🤔 With this form we don't have to create additional connections on the nim module.

In a perfect world the `ChatColumnView` would not need to know about nim modules 😄 And it would help in a future clean-up if we don't add more module dependencies.

Reply:
I'll try. My QML is rusty :D